flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTestHarness.java

```diff
@@ -111,8 +111,8 @@
 	protected StreamTestSingleInputGate[] inputGates;
 
 	public StreamTaskTestHarness(
-			Function<Environment, ? extends StreamTask<OUT, ?>> taskFactory,
-			TypeInformation<OUT> outputType) {
+		Function<Environment, ? extends StreamTask<OUT, ?>> taskFactory,
```

Comment:
Could you revert the whitespace changes? They are questionable for me and even if we want them, they shouldn't be part of the same commit.